gaffer-as-a-service/gaas-rest/src/main/java/uk/gov/gchq/gaffer/gaas/handlers/DeploymentHandler.java

```diff
@@ -54,7 +53,7 @@
 public class DeploymentHandler {
 
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(DeploymentHandler.class);
+    private static final Logger LOGGER = Logger.getLogger(DeploymentHandler.class);
```

Comment:
Why is this being changed from using SLF4J? SLF4J should be used unless a feature is required from Reload4J which does not exist in SLF4J. I don't think this change is required.

Reply:
Reverted